### PR TITLE
Homepage fix

### PR DESCRIPTION
**Pull request by LBHLBasiu**
same changes as per the homepage-design branch. styling bugs fixed

### CODE REVIEW

File: src/pages/home-pages/home-pages.pages.jsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+          <li> I want the ability to view the details of the results returned in a search so that I can easily look through the different results and make the correct selection.</li>
+          <li>I want to be able to register for the application by entering my name, email address, my password and confirming my password so that I can feel secure. </li>
+          <li>As a user I can log in into the application by entering my email and chosen password  so that I can feel my account is being secure </li>
+          <li>As a user I can reset my password if i have forgotten my password </li>
+          <li>As a user I can see my personal information and the APIs used on my profile </li>
+          <li>I can edit my details such as the email address so that I can continue using the same account </li>
+          <li>As a service user I would like to see the list of APIs available in the catalogue so that I can make an informed decision for the product development.</li>
+        </ul>
+
+        <h3>As an Aplication Support Analyst</h3>
```
Comment (humulla) [inline src/pages/home-pages/home-pages.pages.jsx:62]: `Application` typo

File: src/pages/home-pages/home-pages.pages.jsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    </div>
+  );
+};
+
+export const HomePage3 = () => {
+  return (
+    <div className="main-container">
+      <h3>API Specifications </h3>
+      <br/>
+        <a href = "https://playbook.hackney.gov.uk/api-specifications/ ">You can find our API Specification Playbook here</a>
```
Comment (humulla) [inline src/pages/home-pages/home-pages.pages.jsx:107]: Should there be a space at the end of the `"`?

File: src/pages/home-pages/home-pages.pages.jsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        <br/><br/>
+
+    <h3>Why you might need an API</h3>
```
Comment (humulla) [inline src/pages/home-pages/home-pages.pages.jsx:110]: Add `?` at the end

File: src/pages/home-pages/home-pages.pages.jsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  );
+};
+
+export const HomePage4 = () => {
+  return (
+    <div className="main-container">
+        <h3> Our Ways of Working</h3>
+
+        <p>All HackIT developed code should meet the <a href= "https://github.com/LBHackney-IT/Hackney-Development-Standards">12 Hackney Development Standards.</a>
+           around the world to ensure that HackIT APIs are consistent,supportable and of high quality.</p>
```
Comment (humulla) [inline src/pages/home-pages/home-pages.pages.jsx:127]: Around the world?

File: src/pages/home-pages/home-pages.pages.jsx [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+         <p>All newly created HackIT APIs will be developed using the guidelines in the HackIt API playbook.
+            It provides clear guidance on the standards that are being followed when developing new APIs
+            and is also used whenever amending old APIs which were created prior to the existence of the Playbook.
```
Comment (humulla) [inline src/pages/home-pages/home-pages.pages.jsx:131]: `that were created prior ....`

**Comment by LBHLBasiu** [comment]
content updated as per the comments received

**Pull request merged**